integrationtest/org/daisy/dotify/formatter/test/resource-files/marker-ref/marker-ref-display-when-input.obfl

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<obfl version="2011-1" xml:lang="en" xmlns="http://www.daisy.org/ns/2011/obfl">
+	<meta xmlns:dc="http://purl.org/dc/elements/1.1/">
+		<dc:title>Marker reference test for block that is not displayed</dc:title>
+		<dc:description>Tests marker references for a block that is not displayed.</dc:description>
+	</meta>
+	<layout-master name="main" page-width="6" page-height="6" inner-margin="0" outer-margin="0" row-spacing="1" duplex="true">
+		<default-template>
+			<header>
+				<field><marker-reference marker="pagenum" direction="forward" scope="sequence"/></field>
+			</header>
+			<footer/>
+		</default-template>
+	</layout-master>
+	<sequence master="main" initial-page-number="1">
+		<block display-when="(! $starts-at-top-of-page)" keep="page"><marker class="pagenum" value="1"/></block>
```

Comment:
It would be nice to modify this line slightly so that it is clear that this block is not displayed indeed. So please add ⠤⠤:

`<block display-when="(! $starts-at-top-of-page)" keep="page"><marker class="pagenum" value="1"/>⠤⠤</block>`